jbpm-services/jbpm-services-ejb/jbpm-services-ejb-client/pom.xml

```diff
@@ -20,17 +20,6 @@
       <artifactId>jbpm-services-ejb-api</artifactId>
     </dependency>
     
-    <!-- uncomment to test with application deployed on JBoss EAP 6.1 -->
-    <!-- 
-    <dependency>
-      <groupId>org.jboss.as</groupId>
-      <artifactId>jboss-as-ejb-client-bom</artifactId>
-      <version>7.2.0.Final</version>
-      <optional>true</optional>
-      <type>pom</type>
-    </dependency>
-    -->
```

Comment:
+1 to upgrade to wildfly but I'd prefer to keep this as it is simple to use when working on code changes while the container tests are good in builds